astroplan/tests/test_moon.py

```diff
@@ -26,6 +35,8 @@ def test_illumination():
 
     assert_allclose(illumination1, pyephem_illumination, atol=0.05)
 
+
+@pytest.mark.skipif('not HAS_PYEPHEM')
```

Comment:
Remove this pytest mark.
pytest marks are only useful for test functions, and this is not a test function.